MergerLogic/Clients/GpkgClient.cs

```diff
@@ -140,18 +139,11 @@ public override Tile GetTile(int z, int x, int y)
                     command.Parameters.AddWithValue("$z", z);
                     command.Parameters.AddWithValue("$x", x);
                     command.Parameters.AddWithValue("$y", y);
-
-                    var blob = (byte[])command.ExecuteScalar();
-                    if (blob == null)
-                    {
-                        return null;
-                    }
-
-                    tile = new Tile(z, x, y, blob);
+                    blob = (byte[])command.ExecuteScalar();
                 }
             }
 
-            return tile;
+            return this.CreateTile(z, x, y, blob);
```

Comment:
dont return a method (separate to a different line) - it's a bad practice (hard to follow logs and debug later)